### PR TITLE
fix: broken back button in renderPortfolioFormik

### LINKED ISSUE #919: Back button on application page not working
Clicking 'back' on the second application page does not do anything (including bring me to the first page).

### PR DESCRIPTION
**Pull request by jamesxu123**
### Tickets:

- HCK-#919 

### List of changes:

- Back button should work again without breaking next page in `renderEducationFormik()`

### Type of change:

Please delete options that aren't relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How did you do this?
- Added a placeholder reset button that is hidden through CSS
### PR Checklist:

- [x] Merged `develop` branch (before testing)
- [x] Linted my code locally
- [ ] Listed change(s) in the Changelog
- [x] Tested all links in project relevant browsers
- [x] Tested all links on different screen sizes
- [x] Referenced all useful info (issues, tasks, etc)

### Screenshots:

close #919

### CODE REVIEW

File: src/features/Application/ManageApplicationForm.tsx [hunk 1]
```diff
@@ -260,6 +260,16 @@ const ManageApplicationForm: React.FC<IManageApplicationProps> = (props) => {
         </div>
 
         <div className="buttons">
+          <Button
+            type="reset"
+            isLoading={false}
+            disabled={true}
+            variant={ButtonVariant.Secondary}
+            isOutlined={true}
+            style={{ marginRight: '24px', display: 'none' }}
```
Comment (logan-r) [inline src/features/Application/ManageApplicationForm.tsx:269]: nit: remove marginRight

**Review by logan-r — CHANGES_REQUESTED** [review]
Can you add screenshots and confirm this works across chrome, edge, firefox, opera & safari?

**Comment by jamesxu123** [comment]
- [x] Firefox
- [x] Chrome
- [x] Edge Chromium
- [x] Safari
- [x] Opera

image

**Review by logan-r — APPROVED** [review]
👍

**Pull request merged**